packages/reporter/src/hooks/hook-model.spec.ts

```diff
@@ -1,4 +1,5 @@
 import sinon from 'sinon'
+
```

Comment:
nitpick: stray whitespace maybe?

Reply:
I might have added it to separate external dependencies and internal ones. But I think it doesn't matter to remove it.